### ISSUE TITLE
Can't create migrations table

### ISSUE THREAD

**Issue by cliftonc** [guidesmiths/marv-mysql-driver#1]
```
{ Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '(3),
```

It relates to the (3) after the DATETIME: https://github.com/guidesmiths/marv-mysql-driver/blob/master/sql/ensure-migrations-tables.sql#L4

It is only available in MySQL 5.7 and later.  

Can we drop the (3)?

**Comment by cressie176** [comment]
Yep. Thanks for the PR.

**Comment by cressie176** [comment]
Published as marv-mysql-driver@1.2.0
